tests/gtest/aviftilingtest.cc

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 Google LLC. All rights reserved.
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "avif/avif.h"
+#include "avif/internal.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(TilingTest, SetTileConfiguration) {
+  constexpr int kThreads = 8;
+  int tile_rows_log2;
+  int tile_cols_log2;
+  // 144p
+  avifSetTileConfiguration(kThreads, 256, 144, &tile_rows_log2,
+                           &tile_cols_log2);
+  EXPECT_EQ(tile_rows_log2, 0);
+  EXPECT_EQ(tile_cols_log2, 0);
```

Comment:
Optional: To reduce this test's length, consider one of the following helper functions:
```cpp
void TestSetTileConfiguration(int threads, uint32_t width, uint32_t height, int expected_tile_rows_log2, int expected_tile_cols_log2) {
  int tile_rows_log2, tile_cols_log2;
  avifSetTileConfiguration(threads, width, height, &tile_rows_log2, &tile_cols_log2);
  EXPECT_EQ(tile_rows_log2, expected_tile_rows_log2);
  EXPECT_EQ(tile_cols_log2, expected_tile_cols_log2);
}
...
  TestSetTileConfiguration(kThreads, 256, 144, 0, 0);
```
```cpp
std::pair<int, int> GetTileConfig(int threads, uint32_t width, uint32_t height) {
  int tile_rows_log2, tile_cols_log2;
  avifSetTileConfiguration(threads, width, height, &tile_rows_log2, &tile_cols_log2);
  return {tile_rows_log2, tile_cols_log2};
}
...
  EXPECT_EQ(GetTileConfig(kThreads, 256, 144), {0, 0});
```